### PR TITLE
[feature] Open browser automatically when linking, and close CLI after linking or rejecting

### PR DESCRIPTION
**Pull request by cazala**
This PR does 2 things:

- open the browser automaticaly when running `dcl link` (also on `dcl push`)
- print a message and exit the CLI after linking (or if the user rejects the tx) so they don't have to `ctrl-c`

To the reviewer:

1. checkout `feature/open-close-linker`
2. npm i && npm run build
3. npm start -- init && npm -- push

After uploading the scene to IPFS and updating IPNS you the browser should open, prompting you to do a MetaMask transaction. If you reject, that will print a message on the CLI warning that the project was not linked, and exit. If you send the transaction it should print a green message saying the project was linked, and exit the CLI.

### CODE REVIEW

File: src/utils/linker.ts [hunk 1]
```diff
@@ -64,6 +65,17 @@ export async function linker(vorpal: any, args: any, callback: () => void) {
     })
   });
 
+  router.get('/api/close', async (ctx) => {
+    ctx.res.end()
+    const ok = require('url').parse(ctx.req.url, true).query.ok
+    if (ok === 'true') {
+      console.log(chalk.green('\nThe project was linked to Ethereum!'))
```
Comment (michaltakac) [inline src/utils/linker.ts]: Use `vorpal.log` instead of `console.log`.
https://github.com/dthree/vorpal/wiki/API-%7C-vorpal#vorpallogstring-strings

Comment (michaltakac) [inline src/utils/linker.ts]: Use vorpal.log instead of console.log.
https://github.com/dthree/vorpal/wiki/API-%7C-vorpal#vorpallogstring-strings

**Comment by michaltakac** [comment]
Thanks, nice feature!
Works great.

**Comment by cazala** [comment]
@michaltakac done (:

**Pull request merged**